ViteMaDose/Views/CentresList/Cells/CentreActionCell.xib

```diff
@@ -0,0 +1,66 @@
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="none" indentationWidth="10" rowHeight="342" id="3pb-QP-cX9" customClass="CentreActionCell" customModule="ViteMaDose" customModuleProvider="target">
+            <rect key="frame" x="0.0" y="0.0" width="402" height="98"/>
+            <autoresizingMask key="autoresizingMask"/>
+            <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" preservesSuperviewLayoutMargins="YES" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="3pb-QP-cX9" id="HZC-Jz-2x6">
+                <rect key="frame" x="0.0" y="0.0" width="402" height="98"/>
+                <autoresizingMask key="autoresizingMask"/>
+                <subviews>
+                    <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="10" translatesAutoresizingMaskIntoConstraints="NO" id="oum-50-WdU">
+                        <rect key="frame" x="24" y="10" width="354" height="78"/>
+                        <subviews>
+                            <stackView opaque="NO" contentMode="scaleToFill" distribution="equalSpacing" translatesAutoresizingMaskIntoConstraints="NO" id="U2q-w2-gvX">
+                                <rect key="frame" x="0.0" y="0.0" width="354" height="78"/>
+                                <subviews>
+                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Title" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="rtX-2u-NpY">
+                                        <rect key="frame" x="0.0" y="0.0" width="33" height="78"/>
+                                        <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                        <nil key="textColor"/>
+                                        <nil key="highlightedColor"/>
+                                    </label>
+                                    <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="3ej-ox-E3h" userLabel="Filter Button">
+                                        <rect key="frame" x="307" y="0.0" width="47" height="78"/>
+                                        <state key="normal" title="Button"/>
+                                        <buttonConfiguration key="configuration" style="plain" image="list.dash" catalog="system" title=""/>
```

Comment:
Can we use `line.horizontal.3.decrease.circle.fill` please?

Reply:
According to _Xcode_, _line.horizontal.3.decrease.circle.fill_ is deprecate and is replaced by _line.3.horizontal.decrease.circle.fill_ which is unavaiblee prior to iOSS 15.0. Keeping this _SF Symbol_ allows users of "older" versions to have the icon.

Reply:
Okay ! It's done.